amd64/root/start.sh

```diff
@@ -32,13 +32,13 @@ if [ "$DECONZ_VNC_MODE" != 0 ]; then
   if [ ! -e /root/.vnc ]; then
     mkdir /root/.vnc
   fi
-  
+
   # Set VNC password
-  if [ "$DECONZ_VNC_PASSWORD_FILE" != 0 ]; then
-      cat "$DECONZ_VNC_PASSWORD_FILE" | tigervncpasswd -f > /root/.vnc/passwd
-  else
-    echo "$DECONZ_VNC_PASSWORD" | tigervncpasswd -f > /root/.vnc/passwd
+  if [ "$DECONZ_VNC_PASSWORD_FILE" != 0  && -f "$DECONZ_VNC_PASSWORD_FILE" ]; then
+      DECONZ_VNC_PASSWORD=$(cat $DECONZ_VNC_PASSWORD)
```

Comment:
The 😸 `cat` is hunting the wrong `dog` - it should be `DECONZ_VNC_PASSWORD_FILE` here 🐶

Reply:
Good catch 🐟